### PR TITLE
new Madagascar wrapper

### PR DESCRIPTION
**Pull request by mloubout**
Superseed https://github.com/JuliaPackaging/Yggdrasil/pull/6627 with proper branch.

### CODE REVIEW

File: R/RSF/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,1887 @@
...
+script = raw"""
+cd $WORKSPACE/srcdir
+
+# install_license
+install_license src/COPYING.txt
+
+# Build
+cd src/
+
+./configure --prefix=${prefix} CC=$(which gcc) CXX=$(which g++) CAIROPATH=${includedir}/cairo FFMPEGPATH=${includedir}/libavcodec/ AR=$(which ar) CPPPATH=${includedir} LIBPATH=${libdir} BLAS=openblas
```
Comment (giordano) [inline R/RSF/build_tarballs.jl:23]:
```suggestion
./configure --prefix=${prefix} CC="${CC}" CXX="${CXX}" CAIROPATH=${includedir}/cairo FFMPEGPATH=${includedir}/libavcodec/ AR=$(which ar) CPPPATH=${includedir} LIBPATH=${libdir} BLAS=openblas
```

File: R/RSF/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,1887 @@
...
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+# install_license
+install_license src/COPYING.txt
+
+# Build
+cd src/
+
```
Comment (giordano) [inline R/RSF/build_tarballs.jl:22]:
```suggestion
cd $WORKSPACE/srcdir/src

# install_license
install_license COPYING.txt

# Build
```

File: R/RSF/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,1887 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "RSF"
```
Comment (giordano) [inline R/RSF/build_tarballs.jl:5]: Why is this called RSF and not Madagascar?

Comment (mloubout) [reply R/RSF/build_tarballs.jl:5]: The main library is `libdrsf` but can change the name

Comment (giordano) [reply R/RSF/build_tarballs.jl:5]: https://docs.binarybuilder.org/stable/building/#Name
> It should be the name of the upstream package, not for example that of a specific library or executable provided by it, even though they may coincide.

File: R/RSF/build_tarballs.jl (hunk 1 above)
Comment (mloubout) [reply R/RSF/build_tarballs.jl]: Those error as well.

**Pull request merged**